File: dotnet/Source/eg/music/Display.cs
// Copyright (c) 2002 Cunningham & Cunningham, Inc.
// Released under the terms of the GNU General Public License version 2 or later.

using System;

namespace eg.music {
    public class Display : fit.RowFixture {

        public override Type getTargetClass() {
            return typeof(Music);
        }

        public override Object[] query() {
            return MusicLibrary.displayContents();
        }
    }
}

File: imp/ruby/lib/eg/music/display.rb
# Copyright (c) 2002 Cunningham & Cunningham, Inc.
# Released under the terms of the GNU General Public License version 2 or later.

require 'fit/row_fixture'

module Eg
  module Music

    class Display < Fit::RowFixture
      def get_target_class; Music; end
      def query; MusicLibrary.display_contents; end
      # missing parse...
    end

  end
end
